sdk/dashboard/arm-dashboard/_meta.json

```diff
@@ -0,0 +1,8 @@
+{
+  "commit": "468e23d2ac1bd3f3956924eadc2cc881a2b37e73",
+  "readme": "specification/dashboard/resource-manager/readme.md",
+  "autorest_command": "autorest --version=3.7.3 --typescript --modelerfour.lenient-model-deduplication --head-as-boolean=true --license-header=MICROSOFT_MIT_NO_VERSION --generate-test --typescript-sdks-folder=D:\\work\\azure-sdk-for-js ..\\azure-rest-api-specs\\specification\\dashboard\\resource-manager\\readme.md --use=@autorest/typescript@6.0.0-alpha.18.20220329.1 --generate-sample=true",
+  "repository_url": "https://github.com/Azure/azure-rest-api-specs.git",
+  "release_tool": "@azure-tools/js-sdk-release-tools@2.2.1",
+  "use": "@autorest/typescript@6.0.0-alpha.18.20220329.1"
```

Comment:
@colawwj could you update to the latest version for release tool and typescript extension?